ngageoint/opensphere
# feat(lineDash): Add line style options to support a dashed border

In the Add/Edit Place dialog, if a shape is a line or polygon, add controls for styling the line Dash of the stroke 

fix #532

Live Demo: 3d-dash.surge.sh
Test steps:
1. Draw a polygon (square, circle, or polygon) on the map
2. In the style accordion try out various border options
3. Verify that the polygon border changes in the preview as different options are selected
4. Save the polygon and verify that it remembers the dash style when you edit it
5. Draw a line on the map and verify the border options
6. Verify that editing the line reflects the border dash style
7. Verify that the line style looks similar on the the map between 2D and 3D mode
8. In the layers menu expand the Saved Places layer and click on a place
9. Verify that the style accordion correctly shows the border dash style
10. Verify that you can change the dash style
11. Verify that the style works with the command stack (ctrl + z and ctrl + y)
12. Verify that exporting Saved Places as a place preserves the border dash style
13. Export saved places as a kml, verify that it imports with the boarder dash style
14. Select the kml layer in the layers window and in the style accordion check 'replace feature style'
15. Verify that all shapes reflect the selected border dash style
16. Export Saved Places as a csv, verity that it does not import with the border dash style
17. Select the csv layer in the layers window and from the style accordion verify that all shapes reflect the selected border dash style
18. In the layers window, right click on the csv file and from the menu select feature actions
19. Create a feature action that sets the style to have a dashed border
20. Verify that the dashed border changes as expected
21. Verify that exporting/importing the feature action with dashed border style works
22. Refresh Open Sphere and verify that the dash styles are persisted
